Source/WebCore/accessibility/isolatedtree/AXIsolatedObject.cpp

```diff
@@ -1998,6 +1999,13 @@ void AXIsolatedObject::setAccessibleName(const AtomString&)
     ASSERT_NOT_REACHED();
 }
 
+String AXIsolatedObject::textContentPrefixFromListMarker() const
+{
+    if (m_propertyMap.contains(AXProperty::TextContentPrefixFromListMarker))
+        return propertyValue<String>(AXProperty::TextContentPrefixFromListMarker);
+    return { };
```

Comment:
Do we need to check if the map contains this property? Or would it be equivalent (and one less hash map lookup) to do:

```cpp
return propertyValue<String>(AXProperty::TextContentPrefixFromListMarker);
```

If this way works, we could even inline it in the header too (though the compiler would probably inline a one-liner function anyways)

Reply:
You're right, that works. That suggests we can fix some other similar code that calls contains before propertyValue.